sdk/compute/Microsoft.Azure.Management.Compute/src/Microsoft.Azure.Management.Compute.csproj

```diff
@@ -9,19 +9,15 @@
       Provides developers with libraries for the updated compute platform under Azure Resource manager to deploy virtual machine, virtual machine extensions and availability set management capabilities. Launch, restart, scale, capture and manage VMs, VM Extensions and more. Note: This client library is for Virtual Machines under Azure Resource Manager.
       Development of this library has shifted focus to the Azure Unified SDK. The future development will be focused on "Azure.ResourceManager.Compute" (https://www.nuget.org/packages/Azure.ResourceManager.Compute/). Please see the package changelog for more information.
     </Description>
-    <Version>55.0.0</Version>
+    <Version>56.0.0</Version>
```

Comment:
This will be in replaced by the new Compute track 2 sdk

Reply:
what do you mean? doesnt track1 and track2 release in different name space? 
also I thought we are continuing track1 until PowerShell migration to track2 is in place

Reply:
I see, it's still under Compute namespace, then it's ok to release one for PS integration